flash: unable to flash UF2 device using Fedora

**Distro**: Fedora 31
**tinygo version**: 0.11.0

$ **ls -al /dev/ttyACM0** 
crw-rw----. 1 root dialout 166, 0 Mar 14  2020 /dev/ttyACM0

$ **groups**
my_username wheel dialout docker

$ **tinygo flash -target=pybadge main.go**

error: failed to flash /tmp/tinygo_xxxxxx_/main.uf2: unable to locate UF2 device: PYBADGEBOOT

$ **id -a**
uid=_xxxx_(my_username) gid=_xxxx_(my_username) groups=_xxxx_(my_username),10(wheel),_xx_(dialout),971(docker) context=unconfined_(etc. etc.)


I've also tried:
$ **tinygo flash -port=/dev/ttyACM0 -target=pybadge main.go**

However...
I can easily copy UF2 files directly to the PYBADGEBOOT over USB with no issues.

How can I get tinygo flashing to work on Fedora 31?

Not sure if this related, but I get this new error trying to flash the name badge code:
$ **tinygo build -o /run/media/$USER/PYBADGEBOOT/firmware.uf2 -target=pybadge main.go**
# main.go
main.go:25:22: SPI1_MISO_PIN not declared by package machine

Changing the code from MISO to MIS0 doesn't appear to help....

--- a/main.go
+++ b/main.go
@@ -719,7 +719,12 @@ func flashUF2UsingMSD(volume, tmppath string, options *compileopts.Options) erro
 	var infoPath string
 	switch runtime.GOOS {
 	case "linux", "freebsd":
-		infoPath = "/media/*/" + volume + "/INFO_UF2.TXT"
+		_, err := os.ReadDir("/run/media")
+		if err != nil {
+			infoPath = "/media/*/" + volume + "/INFO_UF2.TXT"
+		} else {
+			infoPath = "/run/media/*/" + volume + "/INFO_UF2.TXT"
+		}
 	case "darwin":
 		infoPath = "/Volumes/" + volume + "/INFO_UF2.TXT"
 	case "windows":
@@ -743,7 +748,12 @@ func flashHexUsingMSD(volume, tmppath string, options *compileopts.Options) erro
 	var destPath string
 	switch runtime.GOOS {
 	case "linux", "freebsd":
-		destPath = "/media/*/" + volume
+		_, err := os.ReadDir("/run/media")
+		if err != nil {
+			infoPath = "/media/*/" + volume
+		} else {
+			infoPath = "/run/media/*/" + volume
+		}
 	case "darwin":
 		destPath = "/Volumes/" + volume
 	case "windows":

--- a/main.go
+++ b/main.go
@@ -719,8 +719,8 @@ func flashUF2UsingMSD(volume, tmppath string, options *compileopts.Options) erro
 	var infoPath string
 	switch runtime.GOOS {
 	case "linux", "freebsd":
-		_, err := os.ReadDir("/run/media")
-		if err != nil {
+		fi, err := os.Stat("/run/media")
+		if err != nil || !fi.IsDir() {
 			infoPath = "/media/*/" + volume + "/INFO_UF2.TXT"
 		} else {
 			infoPath = "/run/media/*/" + volume + "/INFO_UF2.TXT"
@@ -748,11 +748,11 @@ func flashHexUsingMSD(volume, tmppath string, options *compileopts.Options) erro
 	var destPath string
 	switch runtime.GOOS {
 	case "linux", "freebsd":
-		_, err := os.ReadDir("/run/media")
-		if err != nil {
-			infoPath = "/media/*/" + volume
+		fi, err := os.Stat("/run/media")
+		if err != nil || !fi.IsDir() {
+			destPath = "/media/*/" + volume
 		} else {
-			infoPath = "/run/media/*/" + volume
+			destPath = "/run/media/*/" + volume
 		}
 	case "darwin":
 		destPath = "/Volumes/" + volume

